src/components/Logo/LogoBlack.js

```diff
@@ -1,12 +1,12 @@
 import React from 'react';
 import classnames from 'classnames';
-import CloudinaryImage from '../CloudinaryImage';
+import { AnimatedLogo } from './AnimatedLogo';
 
 import s from './styles.module.scss';
 
 const LogoBlack = ({ className }) => (
   <div className={classnames(s.logoContainerBlack, className)}>
-    <CloudinaryImage filename="logo-black.svg" className={s.logoBlack} alt="Modus Labs" />
+    <AnimatedLogo s={s} classnames={classnames} />
```

Comment:
it looks like passing `s` property is useless here. I suggest leaving it out together with `classnames`

Reply:
On it